### DISCUSSION TITLE
post/hugo/

### DISCUSSION THREAD

**Discussion by giscus** [SHIERTX/static-blog#1] [Comments]
# post/hugo/

此文为我在使用 Hugo 时乱七八糟的笔记，以 Windows 为例，欢迎参考~

https://www.liczhi.com/post/hugo/

**Comment by cheezhi** [comment]
其实吧，我觉得你原来的动态博客挺不错的啊（买handsome主题的八十多块就这样浪费啦？）
加载速度也不会慢
是什么让你放弃了动态博客了呢（好奇）
话说Hugo支持后台在线写作吗

**Reply by zaochih** [reply]
> 话说Hugo支持后台在线写作吗

~~支持离线写作~~
可以试试 [forestry.io](//forestry.io)
